themes/default/content/_index.md

```diff
@@ -102,22 +102,22 @@ benefits:
       icon: code-window
       icon_color: salmon
       description: |
-        One consistent approach to cloud engineering provides full access to over 50 cloud providers
-        and works for the entire team as you scale up, eliminating accidental complexity.
+        Build with over 60 cloud providers using a consistent, cloud engineering approach that 
+        works for developers and infrastructure teams and eliminates accidental complexity. 
 
     - title: Bring the cloud closer to application development
       icon: download-from-cloud
       icon_color: violet
       description: |
-        Leverage a rich and programmable cloud interface, with reusable packages that abstract
-        away complexity, to bring cloud concepts closer to your application architectures.
+        Leverage a rich, programmable cloud interface and [reusable packages](/product/packages/) that abstract
```

Comment:
@cnunciato this also isn't showing up in the preview properly, do you know how to fix it?